src/lib/stores/workstreams/index.ts

```diff
@@ -55,6 +56,7 @@ export interface EnrichedWorkstream {
   fetchedAt: Date;
   onChainData?: OnChainData;
   data: Workstream;
+  relevant: boolean;
```

Comment:
Could we improve the variable name here? How about `associatedWithLoggedInUser` or something like that?

Reply:
Yes, will do 👍